src/molecules/Product.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Box } from 'atomic-layout'
+import { Text } from '../atoms/Text'
+import { ProductButton } from '../atoms/Button'
+
+const StyledContainerProduct = styled.div`
+  max-width: 500px;
+  height: auto;
+`
+const StyledProductDescription = styled.div`
+  text-align: center;
+  padding: 25px 0;
+`
+const StyledContainerImage = styled.div`
+  position: relative;
+  z-index: 0;
```

Comment:
I don't think this does something. Try removing `z-index` and see if it changes something. If not, `z-index` is not needed.